src/middleware.ts

```
@@ -2,43 +2,45 @@ import { NextResponse } from "next/server";
+  // Define paths that are considered public (accessible without authentication)
+  const isPublicPath = path === "/login";
+
+  // Get the token from the cookies or headers
+  const token = request.cookies.get("auth-storage")?.value;
+
+  // Parse the token to check if user is authenticated
+  let isAuthenticated = false;
+  if (token) {
+    try {
+      const authData = JSON.parse(token);
+      const authData = JSON.parse(token) as {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'authData'. Consider to delete it or rename it.

'authData' is defined here:


(lint/suspicious/noRedeclare)